docs/website/contents/for-developers/Glossary.md

```diff
@@ -330,6 +358,30 @@ Do not select more than `k` blocks past the intersection of all candidate fragme
 
 Motivation: ensure that we can still switch to any candidate chain while respecting the maximum rollback limit.
 
+### :LoE Fragment
+
+The common prefix of all candidate fragments, anchored at the tip of the current chain selection
+
+### :LoE Tip
+
+The tip of the LoE fragment
+
+## :Limit on Patience (LoP)
+
+Disconnect peers that advertise a better header than their current one, but haven't provided it quickly enough.
+
+Motivation: because of the [LoE](#limit-on-eagerness-loe), we don't select headers past `k` blocks beyond the common intersection of all candidate fragments. Therefore, an attacker could prevent a syncing node from making progress by stalling indefinitely at an intersection, promising headers to extend their chain but never delivering them. The LoP mitigates that attack.
+
+In the current version of Genesis, the limit on patience is implemented by a "leaky token bucket" algorithm.
+
+### :TCAP
```

Comment:
I don't know if it's obvious that TCAP stands for Token Capacity; maybe worth adding?
```suggestion
### :TCAP — Token capacity
```